helper/testhelpers/testhelpers.go

```diff
@@ -623,7 +623,7 @@ func GenerateDebugLogs(t testing.T, client *api.Client) chan struct{} {
 	return stopCh
 }
 
-func VerifyRaftPeers(t testing.T, client *api.Client, expected map[string]bool) {
+func VerifyRaftPeers(t testing.T, client *api.Client, expected map[string]bool) error {
```

Comment:
I am looking at this code for the first time. I see that "expected" map has values like "core-0: true". What do the boolean values represent here ?

Reply:
Good call! I'll go ahead and add a comment here to explain for first-time consumers.